extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -164,25 +187,24 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   onLayoutMeasure() {
-    /** @private {number} */
     this.slideWidth_ = this.getLayoutWidth();
 
-    /** @private {number} */
     this.previousScrollLeft_ = this.slidesContainer_./*OK*/scrollLeft;
   }
 
   /** @override */
   layoutCallback() {
-    if (this.slideIndex_ == null) {
+    if (this.slideIndex_ === null) {
```

Comment:
this is because it will always be false based on the [Abstract Equality Comparison Algorithm](http://es5.github.io/#x11.9.3) cause there is no rule for it